config/translations/console.fr.yml

```diff
@@ -199,41 +199,41 @@ commands:
                 module-path: 'Entrez le chemin du module'
                 description: 'Entrez la description du module'
                 core: 'Entrez la version core Drupal'
-                package: 'Entrez le nom du paquet'
+                package: 'Entrez le nom du paquet (<info>package</info>)'
                 controller: 'Voulez-vous générer un contrôleur par défaut'
-                composer: 'Do you want to add a composer.json file to your module'
-                dependencies: 'Would you like to add module depencencies'
-                test: 'Voulez-vous générer une classe de test unitaire'
+                composer: 'Voulez-vous générer un fichier composer.json file pour votre module ?'
+                dependencies: 'Voulez-vous ajouter des dépendances à votre module ?'
+                test: 'Voulez-vous générer une classe de test unitaire ?'
             warnings:
-                module-unavailable: 'Warning The following modules are not already available in your local environment "%s"'
+                module-unavailable: 'Attention, les modules suivants ne sont pas disponible dans votre installation locale : "%s"'
             errors:
-                directory-exists: 'The target directory "%s" is not empty.'
+                directory-exists: 'Le dossier cible "%s" n''est pas vide'
         theme:
-            description: 'Generate a theme.'
-            help: 'The <info>generate:theme</info> command helps you generates a new theme.'
-            welcome: 'Welcome to the Drupal theme generator'
+            description: 'Générer un theme.'
+            help: 'La commande <info>generate:theme</info> vous assiste dans la création d''un nouveau thème.'
+            welcome: 'Bienvenue dans le générateur de thème pour Drupal'
             options:
-                theme: 'The theme name'
-                machine-name: 'The machine name (lowercase and underscore only)'
-                theme-path: 'The path of the theme'
-                description: 'Theme description'
-                core: 'Core version'
-                package: 'Theme package'
-                composer: 'Add a composer.json file'
-                base_theme: 'Base theme (i.e. classy, seven)'
+                theme: 'Nom du thème'
+                machine-name: 'Le nom machine (minuscules et underscore _ seulement)'
+                theme-path: 'Dossier du thème'
+                description: 'Description du thème'
+                core: 'Version du core'
```

Comment:
I believe it is customary in the French  Drupal community to talk about "core". I would keep it as such.